pkg/storegateway/bucket.go

```diff
@@ -840,10 +840,8 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 
 	var (
 		ctx              = srv.Context()
-		stats            = &queryStats{}
+		stats            *queryStats
```

Comment:
is this declaration necessary? It just gets re-declared further down

Reply:
this should have been fixed with https://github.com/grafana/mimir/pull/3540/commits/0f75572f18f97f63f2e207e92bb1818fdfe1cca9